source/administration/changelog.md

```diff
@@ -2,6 +2,119 @@
 
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
+## Release v3.4.0
+
+Expected release date: 2016-08-16
+
+### Highlights
+
+Heroku Deployment
+
+Zapier App
```

Comment:
In this highlight maybe also mention Auth 2.0 provider, which enables secure auth into 3rd party applications. This was in 3.3 but it wasn't explained.